src/vstart.sh

```diff
@@ -510,14 +510,6 @@ wconf() {
     fi
 }
 
-get_pci_selector() {
-    which_pci=$1
-    lspci -mm -n -D -d $pci_id | cut -d ' ' -f 1 | sed -n $which_pci'p'
-}
-
-get_pci_selector_num() {
-    lspci -mm -n -D -d $pci_id | cut -d' ' -f 1 | wc -l
-}
```

Comment:
why do you think by removing these helpers we can make it easier to choose a PCI device?

Reply:
Those helper  make vendor::device to pci bus .  i specify these parameters directly.

Reply:
@majianpeng my point is that this change does not make the life easier if one *have to* specify them directly.

Reply:
@tchaikov .  Like parameter --bluestore-devs, we set the device name directly.  Before vstart.sh, we should call spdk/scritps/setup.sh. we can get all infos of nvme, like:

>>0000:81:00.0 (8086 2701): Skipping un-whitelisted NVMe controller nvme0n1
>>0000:b5:00.0 (8086 0a54): nvme -> uio_pci_generic
>>0000:b6:00.0 (8086 0a54): nvme -> uio_pci_generic
>>0000:b7:00.0 (8086 0a54): nvme -> uio_pci_generic
>>0000:b8:00.0 (8086 0a54): nvme -> uio_pci_generic
From that, we know all nvme pci bus info .